src/components/Admin/ManagementPageTemplate.tsx

```diff
@@ -17,16 +17,33 @@ interface Props extends PageProps {
       url: string;
     }>;
   }>;
+  title?: string;
+  returnUrl?: string;
```

Comment:
This must represent that it's not actually a return URL, but rather the entity's page URL imo.